main.js

```diff
@@ -0,0 +1,16 @@
+const user = new APIManager
+const renderer = new Renderer
+
+$("#load").on("click", () => {
+    user.userData()
```

Comment:
Better to have a method inside the APIManager class that's job is to call all the internal methods. Therefore the controller isn't controlling that, rather it is an internal operation.